tests/torch/sparsity/movement/helpers/utils.py

```diff
@@ -37,7 +37,10 @@ def mock_linear_nncf_node(
 ) -> NNCFNode:
     graph = NNCFGraph()
     linear = graph.add_nncf_node(
-        node_name, "linear", Mock(), LinearLayerAttributes(True, in_features, out_features, bias=bias)
+        node_name,
+        "linear",
+        Mock(),
+        LinearLayerAttributes(True, in_features=in_features, out_features=out_features, bias=bias),
```

Comment:
Why did you not add name for first argument `weight_requires_grad`?

Reply:
Done